introduction.md

```diff
@@ -0,0 +1 @@
+I am Subramanyam S, making commit to get to next challenge in Mastering Github by CodeSchool.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.